resources/styleheets/main.scss

```diff
@@ -0,0 +1,18 @@
+
+//Abstracts 
+@import 'abstracts/variables'; 
+
+//base
+@import 'base/base';
+
+//layout
+@import
+    'layout/header',
+    'layout/invi';
```

Comment:
decide which structure you use and stick to it
for import in a line is valid use 
```
  @import
    'folder/file';
```